lib/cocina/models/validators/description_values_validator.rb

```diff
@@ -51,10 +52,16 @@ def validate_obj(obj, path)
           validate_array(obj, path) if obj.is_a?(Array)
         end
 
-        def validate_values(hash, path)
+        def validate_values_for_blanks(hash, path)
+          return unless hash[:value] && hash[:value].is_a?(String) && hash[:value].blank? && !hash[:value].empty? # rubocop:disable Style/SafeNavigation
```

Comment:
@jcoyne I had to switch from your suggested regex to this `.blank? && !.empty?` because it was catching values that had consecutive newline characters (thus a begin-line/new-line/end-line match happens within a non blank string. If you have a better suggestion than what I have here, let me know.

Reply:
I think I had anchored my regex with `^` and `$`, but I should have used `\A` and `\z` instead the former are end of line and the latter are end of string.